src/components/tidslinje/TidslinjeMedArbeidsgiver.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import TidslinjeElement from './TidslinjeElement';
+import { Normaltekst } from 'nav-frontend-typografi';
+import sykmeldtHvaNa from './svg/sykmeldt-hva-na.svg';
+import innen4uker from './svg/medArbeidsgiver/innen-4uker.svg';
+import innen7uker from './svg/medArbeidsgiver/innen-7uker.svg';
+import innen8uker from './svg/medArbeidsgiver/innen-8uker.svg';
+import innen26uker from './svg/medArbeidsgiver/innen-26uker.svg';
+import innen39uker from './svg/medArbeidsgiver/innen-39uker.svg';
+import sluttfasen3 from './svg/sluttfasen-3.svg';
+
+const TidslinjeMedArbeidsgiver: React.FC = () => (
+    <>
+        <TidslinjeElement erEkspanderbar={false} erForsteElement={true} tittel={'Første sykmeldingsdag'} />
+        <TidslinjeElement
+            erEkspanderbar={true}
```

Comment:
Det holder å skrive "erEkspanderbar" uten "=", da dette sendes inn som en boolean.

I tillegg er det ryddigere å la være å sende "erEkspanderbar" i stedet for å sende den som "false", og heller sette propen til optional i komponenten.

Reply:
Enig der! fikser